packages/api/amplify_api/example/tool/pull_test_backend.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+APP_ID=$API_APP_ID ../../../../build-support/pull_backend_by_app_id.sh 
```

Comment:
Is this needed? Do we really want to make this in every package?

Reply:
Hmm yeah, I see. I'm sure there's a better way, but we don't need to block on it.